### PR TITLE
SATELLITE: Display NaviGator on satellite imagery

### LINKED ISSUE #290: Display NaviGator over satellite imagery
NaviGator has a GPS and absolute (earth fixed) orientation sensors, so it is possible to display NaviGator's current position (and potentially other detected objects) over satellite imagery from open street maps or w/e. Not only would it look really cool, but it would help understand where we are at test days / competition when we can't see NaviGator easily!

* Google if there is some  easy way to do this in ROS already with an rviz / rqt plugin
* Otherwise, research some options to roll this out ourselves (web app is okay)

### PR DESCRIPTION
**Pull request by thedpws**
closes #290 

To launch: `roslaunch rviz_satellite launch.launch`
This package (rviz_satellite) subscribes to `/odom` and `/lla` to display NaviGator on satellite imagery in rviz.

The launch file will launch rviz with the `satellite.rviz` configuration, as well as the node for converting and publishing to `/lla`. 

`satellite.rviz` is based on `navigator.rviz`, but has **all elements scaled up by a factor of 20.**

### CODE REVIEW

**Comment by thedpws** [comment]
image
Results when launched with wholecourse.bag playing.

File: satellite/launch notes [hunk 1]
```diff
@@ -0,0 +1,4 @@
+1. point_stamped_and_odom_to_navsatfix
+2.rviz_satellite demo.launch
+3. should be it
+4. rviz open the config
```
Comment (ironmig) [inline satellite/launch notes]: these instructions should just go into the readme (I also think they're inaccurate, as the file is launch.launch)

**Comment by ironmig** [comment]
You also have some merged conflicts:

Fetch latest master branch:
`git fetch upstream`
Rebase:
`git rebase upstream/master`
The merge conflcit on navgiator.rviz will appear. I recommend starting with the one on the repo, adding your plugin, then continuing.
`git checkout upstream/master navigator.rviz`
Open it in rviz and add your plugin
`git add navigator.rviz`
`git rebase --continue`

**Comment by ironmig** [comment]
I went ahead and did the rebase for you. You can make the further changes off this:
`git fetch upstream`
`git checkout upstream/sat -B master`
`git push <your remote> master -f`

Then make further commits and push them to.

**Comment by ironmig** [comment]
Can't figure out why CI is failing, seems unrelated to your PR so I'll merge.

**Pull request merged**